float64 razdalja
time cas
